Controller/Adminhtml/Reindex/Save.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * Module Algolia Algoliasearch
+ */
+namespace Algolia\AlgoliaSearch\Controller\Adminhtml\Reindex;
+
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\App\Action\Context;
+use Magento\Catalog\Model\ProductFactory;
+use Magento\Store\Model\StoreManagerInterface;
+use Algolia\AlgoliaSearch\Helper\Data as DataHelper;
+use Algolia\AlgoliaSearch\Helper\Entity\ProductHelper;
+use Algolia\AlgoliaSearch\Exception\AlgoliaReindexException;
+use Algolia\AlgoliaSearch\Exception\AlgoliaProductDisabledException;
+use Algolia\AlgoliaSearch\Exception\AlgoliaProductDeletedException;
+use Algolia\AlgoliaSearch\Exception\AlgoliaProductNotVisibleException;
+use Algolia\AlgoliaSearch\Exception\AlgoliaProductOutOfStockException;
+
+/**
+ * Class: Save
+ */
+class Save extends \Magento\Backend\App\Action
+{
+
+    const MAX_SKUS = 10;
+
+    /**
+     * @var ProductFactory
+     */
```

Comment:
Could you "one-line" those annotations to `/** @var ProductFactory */`?
The same applies to all one line annotations / comments :)